<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActiveMQModuleWidget</class>
 <widget class="QWidget" name="ActiveMQModuleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ActiveMQModuleWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="mBrokerURILabel_">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;BrokerURI&lt;span style=&quot; font-weight:600;&quot;&gt; : &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="mBrokerURIEdit_">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>failover:(tcp://localhost:61616/)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="mIsUseTopicsCheckBox_">
         <property name="text">
          <string>useTopics</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="mSessionTransactedCheckBox_">
         <property name="text">
          <string>sessionTransacted</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_topic_persistent">
       <property name="title">
        <string>主题持久订阅</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="mClientIdLabel_">
          <property name="text">
           <string>clientID：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="mCLientIdEdit_">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="mSubNameLabel_">
          <property name="text">
           <string>subscriptionName：</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="mSubNameEdit_">
          <property name="text">
           <string>4028c7f843710b7b0143710cc8070004</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="mTopicNameLabel_">
       <property name="text">
        <string>TopicName：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="mTopicNameEdit_">
       <property name="text">
        <string>topic://NoticeTopic</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="mUserNameLabel_">
       <property name="text">
        <string>UserName：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_username"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="mPassWordLabel_">
       <property name="text">
        <string>Password：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_password">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="mQueueName_">
       <property name="text">
        <string>QueueName</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="mQueueNameEdit_">
       <property name="text">
        <string>queue://QueueTopic</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="mSelectorLabel_">
       <property name="text">
        <string>消息过滤器：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="mSelectorEdit_">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string>注：SQL92语法</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="mReceivedMsgList_">
     <property name="movement">
      <enum>QListView::Static</enum>
     </property>
     <property name="isWrapping" stdset="0">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="mOpenMQBtn_">
       <property name="text">
        <string>打开MQ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mCloseMQBtn_">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>关闭MQ</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="mTestSendMsgToServerBtn_">
     <property name="text">
      <string>测试按钮(发送消息)</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
